chains/solana/contracts/programs/ccip-router/src/instructions/v1/offramp.rs

```diff
@@ -562,8 +561,13 @@ fn get_token_accounts_for<'a>(
 // should_execute_messaging checks if:
 // 1. There is at least one account used for messaging (the first subset of accounts). This is because the first account is the program id to do the CPI
 // 2. AND the logic_receiver has a value different than zeros
```

Comment:
```suggestion
// There is at least one account used for messaging (the first subset of accounts). This is because the first account is the program id to do the CPI
```